board/linux-raspberrypi0.config

```diff
@@ -15,9 +15,12 @@ CONFIG_IKCONFIG=m
 CONFIG_IKCONFIG_PROC=y
 CONFIG_MEMCG=y
 CONFIG_BLK_CGROUP=y
+CONFIG_CFS_BANDWIDTH=y
```

Comment:
See my previous comment, we don't need all these additional settings and modules

Reply:
Can we look at trying to get this merged? It seems like we've hit a bit of a brick wall with the kernel configs, I'm happy to have a crack at reducing the pi4 config as there is a lot of room for improvement there. It will take some help from you guys to find a good solution here as I'm not too familiar with the Raspberry Pi kernel config.

Reply:
As I said many times already, I don't think the overlay approach is the right one. The changes needed would be quite simple, actually:

- Diff the current Pi0/w and Pi4 configs and look for the same lines. These will go into `linux-base.config`
- Put all other configs into their respective `board/linux-${BOARDNAME}.config` config

Ideally, the resulting config for the Pi0/w should stay the same, even with this PR merged.